client/api_bucket.go

```diff
@@ -398,18 +400,32 @@ func (m *GfSpListBucketsByIDsResponse) UnmarshalXML(d *xml.Decoder, start xml.St
 func (c *client) ListBuckets(ctx context.Context, opts types.ListBucketsOptions) (types.ListBucketsResult, error) {
 	params := url.Values{}
 	params.Set("include-removed", strconv.FormatBool(opts.ShowRemovedBucket))
+
+	account := opts.Account
+	if account == "" {
+		acc, err := c.GetDefaultAccount()
+		if err != nil {
+			log.Error().Msg(fmt.Sprintf("failed to get default account:  %s", err.Error()))
+			return types.ListBucketsResult{}, err
+		}
+		account = acc.GetAddress().String()
+	}
+
```

Comment:
check if it is a valid account. .  for example ;  if account != "" {
	_, err := sdk.AccAddressFromHexUnsafe(account )
	if err != nil {
		return "", err
	}
}